helpers/handlebars/helpers/index.js

```diff
@@ -177,6 +187,12 @@ const helpers = () => ({
 		return options.inverse(this);
 	},
 	userIds: (users) => (users || []).map((user) => user._id).join(','),
+	getVersionedPath: (assetPath) => {
```

Comment:
this is just getAssetPath, if it's versioned or not just will be changed inside.
```suggestion
	getAssetPath: (assetPath) => {
```

Reply:
it versioned in a sense that it contains the fingerprint. so no server, but rather a hash..

Reply:
but I would be ok with renaming